yodeploy/hooks/nginx.py

```diff
@@ -0,0 +1,56 @@
+"""Application hooks for apps that use nginx."""
+
+import logging
+import os
+import subprocess
+import sys
+
+from yodeploy.hooks.configurator import ConfiguratedApp
+from yodeploy.util import touch
+
+log = logging.getLogger(__name__)
+
+
+class Nginx(object):
+    @staticmethod
+    def reload():
+        try:
+            subprocess.check_call(('service', 'nginx', 'reload'))
+        except subprocess.CalledProcessError:
```

Comment:
It would have been nice to see more details here. Like this:
```
try:
   subprocess.check_output(cmd, stderr=subprocess.STDOUT)
except subprocess.CalledProcessError as e:
    log(e.returncode, e.output)
```

Reply:
+1